sda/host.go

```diff
@@ -612,3 +621,31 @@ func (h *Host) Rx() uint64 {
 func (h *Host) Address() string {
 	return h.workingAddress
 }
+
+// GetStatus is a function that returns the status report of the server.
+func (h *Host) GetStatus() Status {
+	m := make(map[string]string)
+	nbr := len(h.connections)
+	remote := make([]string, nbr)
+	iter := 0
+	var rx uint64
+	var tx uint64
+	var local string
+	for _, c := range h.connections {
+		remote[iter] = c.Remote()
+		local = c.Local()
+		rx += c.Rx()
+		tx += c.Tx()
+		iter = iter + 1
+	}
+	m["Connections"] = strings.Join(remote, "\n")
+	m["Host"] = local
```

Comment:
OK, I see. This is wrong then.

`m["Host'] = h.Address()`
(or is it `h.ServerIdentity.Address`?)